src/PlaylistController.js

```diff
@@ -0,0 +1,66 @@
+// import assert from 'assert';
+import GenericController from './GenericController';
+import Playlist from './Structs/Playlist';
+
+export default class PlaylistController extends GenericController {
+  constructor(...args) {
+    super(...args);
+    if (window.APPCONTEXT && window.APPCONTEXT.Go && window.APPCONTEXT.Go.h && window.APPCONTEXT.Go.h.length) {
+      this._playlists = Object.assign({}, window.APPCONTEXT.Go.h[0].Hi);
+      this._watchPlaylistObject();
+    } else {
+      this._playlists = {};
+      console.error('Could not listen to playlist changes in the expected location'); // eslint-disable-line
```

Comment:
The idea behind it was I wanted it to fail without breaking anything else.  For instance if this threw an error, the entire `GMusicUI` setup process would fail (this constructor is called during setting up).

If something isn't working and the developer doesn't check console, that is the developers fault not ours.

Also I believe sentry hooks into the standard `console.warn` and `console.error` (someone told me that once) :laughing: